package.json

```diff
@@ -75,6 +74,7 @@
     "browser-process-platform": "~0.1.1",
     "chai": "^4.1.2",
     "cross-env": "^5.1.6",
+    "detect-node": "^2.0.3",
```

Comment:
Note to self: this needs to be moved back into dependencies before merging as it's used in the main code